taskcluster/async/asyncclient.py

```diff
@@ -171,6 +171,8 @@ def _createSession(self):
                         message = "Authentication Error"
                     elif status == 500:
                         message = "Internal Server Error"
+                    else:
+                        message = "Unknown Server Error %s\n%s" % (str(status), data)
```

Comment:
I'd rather that we cap this at, say, 1024 characters.  My concern is that an extremely large response happens to parse as valid JSON data (so as not to raise in the underlying `json.loads`) and utterly spams stdout.